src/backend/distributed/sql/udfs/set_password_if_role_exists/9.1-1.sql

```diff
@@ -0,0 +1,11 @@
+CREATE OR REPLACE FUNCTION set_password_if_role_exists(
+    role_name regrole,
```

Comment:
If we use `regrole`, that'd defeat our purpose for `skip if user not exists`, because Postgres would automatically error out even before our code is called:
```SQL
SELECT set_password_if_role_exists('r2', 'md55ed7902b70147c4a6fb4517eaa0f24b7');;
ERROR:  role "r2" does not exist
LINE 1: SELECT set_password_if_role_exists('r2', 'md55ed7902b70147c4...

```